src/tools/support_documentation_tools.py

```
@@ -0,0 +1,2298 @@
+            perf_ctx.start_phase("search")
+
+            # Perform search with enhanced parameters
+            if search_type == "hybrid":
+                # Use direct database call for hybrid search with custom weights
+                perf_ctx.start_phase("embedding")
+                query_embedding = await embeddings_service.create_embedding(enhanced_query)
+                perf_ctx.end_phase()
+                perf_ctx.start_phase("search")
+
+                # Prepare filter parameters
+                filter_params = support_service._prepare_search_filters(filters)
```

Pylint (3.3.7)

[warning] Access to a protected member _prepare_search_filters of a client class

(W0212)